MediaBrowser.sln

```diff
@@ -1,4 +1,4 @@
-Microsoft Visual Studio Solution File, Format Version 12.00
+﻿Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
encoding change?

Reply:
IDK, done by dotnet itself, this doesn't get touched by hand